denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -0,0 +1,45 @@
+armor_degradation:
+  type: world
+  debug: false
+  events:
+    on player damaged by slime:
+      - if <context.damager.is_mythicmob> && <context.damager.MythicMob.level.is[MORE].to[5]>:
+          - determine passively cancelled
+          - choose <util.random.int[1].to[4]>:
+              - case 1
+                - if <player.equipment_map.get[helmet].material.name> == *_helmet|turtle_helmet
+                  - define slot 40
+                  - define item <player.equipment_map.get[helmet].material.name>
+                  - hurt <context.damager.mythicmob.level.multiply[3]>
+                - else
+                  - hurt <context.damager.mythicmob.level.multiply[5]>
+                  - narrate "<&e>Acid splashes on you, burning your face."
+              - case 2
+                - if <player.equipment_map.get[chestplate].material.name> == *_chestplate
+                  - define slot 39
+                  - define item <player.equipment_map.get[helmet].material.name>
+                  - hurt <context.damager.mythicmob.level.multiply[3]>
+                - else
+                  - hurt <context.damager.mythicmob.level.multiply[5]>
+                  - narrate "<&e>Acid splashes on you, burning your chest."
+              - case 3
+                - if <player.equipment_map.get[leggings].material.name> == *_leggings
+                  - define slot 38
+                  - define item <player.equipment_map.get[helmet].material.name>
+                  - hurt <context.damager.mythicmob.level.multiply[3]>
+                - else
+                  - hurt <context.damager.mythicmob.level.multiply[5]>
+                  - narrate "<&e>Acid splashes on you, burning your legs."
+              - case 4
+                - if <player.equipment_map.get[boots].material.name> == *_boots
```

Comment:
This will return invalid if the map returns that the boots is not in the map.
Missing a colon at the end of the `if` statement
This isn't quite how `==` logic works; regex doesn't go here. optimally, it's best to avoid regex regardless. you want to use `equipment_map[boots].material.name.ends_with[stuff]` here.
```suggestion
                - if <player.equipment_map.contains[boots]> <player.equipment_map.get[boots].material.name.ends_with[_boots]>:
```